assets/script.js

```diff
@@ -84,4 +84,78 @@ function getStreamingData(movieId) {
     });
 }
 
-// getStreamingData("tt0167260");
\ No newline at end of file
+// getStreamingData("tt0167260");
+
+
+// Gets movie data
+function getMovieData(movieId) {
+    var imdbApiUrl = "https://imdb-api.com/en/API/Title/k_zns86b2w/" + movieId;
+
+    fetch(imdbApiUrl).then(function (response) {
+        if (response.ok) {
+            response.json().then(function (data) {
+                console.log("--IMDb--");
+                console.log(data);
+                console.log(data.fullTitle);
+
+                let movieObject = {
+                    title: data.fullTitle,
+                    year: data.year,
+                    rating: data.imDbRating,
+                    image: data.image,
+                    plot: data.plot,
+                    streaming: getStreamingData(data.id)
+                };
+
+                console.log(movieObject);
+
+                return;
+            });
+        }
+    });
+}
+
+// add movie data to "movie-container"
+function addMovieData(movieObject) {
+    let movieContainer = document.querySelector("#movie-container");
+    let movieCard = document.createElement("div");
```

Comment:
There's a movie description div that the title, rating, plot summary, and streaming services (basically anything that's not the image) will go in.